source/math/FloatConvexHull2D.ooc

```diff
@@ -21,55 +21,122 @@ import FloatTransform2D
 import text/StringTokenizer
 
 FloatConvexHull2D: class {
-	// Points must be in clockwise order
-	// TODO: Implement computation of convex hull from a set of points
 	_points: VectorList<FloatPoint2D>
+	_hull: VectorList<FloatPoint2D>
 	points ::= this _points
+	hull ::= this _hull
 	count ::= this _points count
```

Comment:
But you're copying `points` to `hull` if `computeHull` is `false` so nothing would break, right?